### PR TITLE
Baybayin Words, Pixel Number Base JSON Spelling Fixes

### PR DESCRIPTION
**Pull request by Mirage-Xel**

### CODE REVIEW

File: HTML/Baybayin Words.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			   <tr><td>bahay</td><td>house</td><td>kapitan</td><td>captain</td><td>tanso</td><td>copper</td></tr>
+			   <tr><td>pagkain</td><td>food</td><td>palakpak</td><td>clap</td><td>pilak</td><td>silver</td><tr>
+			   <tr><td>tulog</td><td>sleep</td><td>mata</td><td>eyes</td><td>patay</td><td>dead</td></tr>
+			   <tr><td>tulay</td><td>bridge</td><td>sisid</td><td>dive</td><td>bingi</td><td>deaf</td></tr>
+			   <tr><td> kahoy</td><td>wood</td><td>halaman</td><td>plant</td><td>bulag</td><td>blind</td></tr>
+			   <tr><td>asukal</td><td>sugar</td><td>gamot</td><td>medicine</td><td>pilay</td><td>lame</td></tr>
+			   <tr><td>mayaman</td><td>rich</td><td>isla</td><td>island</td><td>pipi</td><td>mute</td></tr>
+			   <tr><td>sapatos</td><td>shoe</td><td>tago</td><td>hide</td><td>mantsa</td><td>stain</td></tr>
+			   <tr><td>ginto</td><td>gold</td><td>langoy</td><td>swim</td><td>plato</td><td>plate</td></tr>
+               <tr><td>kayumanggi</td><td>brown</td><td>lipad</td><td>fly</td><td>kutsilyo</td><td>knife</td></tr>
```
Comment (MrPeanut1028) [inline HTML/Baybayin Words.html:41]: Perhaps fix these two lines that aren't with the others.

File: HTML/img/Component/Baybayin Words.svg [hunk 2]
```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (MrPeanut1028) [inline HTML/img/Component/Baybayin Words.svg:1]: You'll need to put this file through [SVG minification](https://jakearchibald.github.io/svgomg/).

File: JSON/Baybayin Words.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "Author": "Panoptes (Xel), CrunchyBot",
```
Comment (MrPeanut1028) [inline JSON/Baybayin Words.json:2]: Remove either the Authors or the Contributors section, you do not need both.

File: JSON/Pixel Number Base.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
-  "Author": "Panoptes (Xel), Crunchybot",
```
Comment (MrPeanut1028) [inline JSON/Pixel Number Base.json:2]: Just checked the repo. It doesn't actually display it twice but it does show a little `?` when you hover, which isn't supposed to happen. If you could remove this line, that would be great.

File: HTML/Baybayin Words.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <title>Baybayin Words — Keep Talking and Nobody Explodes Module</title>
+    <meta name="viewport" content="initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" type="text/css" href="css/font.css">
```
Comment (samfundev) [inline HTML/Baybayin Words.html:10]:
```suggestion
    <meta charset="utf-8">
    <meta name="viewport" content="initial-scale=1">
    <title>Baybayin Words — Keep Talking and Nobody Explodes Module</title>
    <link rel="stylesheet" type="text/css" href="css/font.css">
    <link rel="stylesheet" type="text/css" href="css/normalize.css">
    <link rel="stylesheet" type="text/css" href="css/main.css">
```

File: HTML/Baybayin Words.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                <img src="img/Component/Baybayin Words.svg" class="diagram">
+                <h2>On the Subject of Baybayin Words</h2>
+                <p class="flavour-text">We don't use this alphabet in our country. Especially since I'm not Filipino.</p>
+
+               <p>The module consists of a display and 6 buttons.</p>
+               <p>The display shows a Tagalog word that is written in Modern Baybayin Alphabet. Translate the word that is the given display into the English alphabet using the Translation Table and locate the English word that matches the given word. Select the translated word in one of the buttons in the module.</p>
+               <p> Choosing the wrong word will cause a strike to occur and the module to reset. </p>
+			   <table>
+			   <tr><td colspan="6"><strong>Translation Table</strong></td></tr>
+			   <tr> <td>Word</td><td>Translation</td><td>Word</td><td>Translation</td><td>Word</td><td>Translation</td></tr>
```
Comment (samfundev) [inline HTML/Baybayin Words.html:30]:
```suggestion
               <table>
               <tr><td colspan="6"><strong>Translation Table</strong></td></tr>
               <tr> <td>Word</td><td>Translation</td><td>Word</td><td>Translation</td><td>Word</td><td>Translation</td></tr> 
```

File: HTML/Baybayin Words.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+               <tr><td>kalapati</td><td>dove</td><td>salamat</td><td>thanks</td><td>damit</td><td>clothing</td></tr>
+			   <tr><td>bahay</td><td>house</td><td>kapitan</td><td>captain</td><td>tanso</td><td>copper</td></tr>
+			   <tr><td>pagkain</td><td>food</td><td>palakpak</td><td>clap</td><td>pilak</td><td>silver</td><tr>
+			   <tr><td>tulog</td><td>sleep</td><td>mata</td><td>eyes</td><td>patay</td><td>dead</td></tr>
+			   <tr><td>tulay</td><td>bridge</td><td>sisid</td><td>dive</td><td>bingi</td><td>deaf</td></tr>
+			   <tr><td> kahoy</td><td>wood</td><td>halaman</td><td>plant</td><td>bulag</td><td>blind</td></tr>
+			   <tr><td>asukal</td><td>sugar</td><td>gamot</td><td>medicine</td><td>pilay</td><td>lame</td></tr>
+			   <tr><td>mayaman</td><td>rich</td><td>isla</td><td>island</td><td>pipi</td><td>mute</td></tr>
+			   <tr><td>sapatos</td><td>shoe</td><td>tago</td><td>hide</td><td>mantsa</td><td>stain</td></tr>
+			   <tr><td>ginto</td><td>gold</td><td>langoy</td><td>swim</td><td>plato</td><td>plate</td></tr>
```
Comment (samfundev) [inline HTML/Baybayin Words.html:40]:
```suggestion
               <tr><td>bahay</td><td>house</td><td>kapitan</td><td>captain</td><td>tanso</td><td>copper</td></tr>
               <tr><td>pagkain</td><td>food</td><td>palakpak</td><td>clap</td><td>pilak</td><td>silver</td><tr>
               <tr><td>tulog</td><td>sleep</td><td>mata</td><td>eyes</td><td>patay</td><td>dead</td></tr>
               <tr><td>tulay</td><td>bridge</td><td>sisid</td><td>dive</td><td>bingi</td><td>deaf</td></tr>
               <tr><td> kahoy</td><td>wood</td><td>halaman</td><td>plant</td><td>bulag</td><td>blind</td></tr>
               <tr><td>asukal</td><td>sugar</td><td>gamot</td><td>medicine</td><td>pilay</td><td>lame</td></tr>
               <tr><td>mayaman</td><td>rich</td><td>isla</td><td>island</td><td>pipi</td><td>mute</td></tr>
               <tr><td>sapatos</td><td>shoe</td><td>tago</td><td>hide</td><td>mantsa</td><td>stain</td></tr>
               <tr><td>ginto</td><td>gold</td><td>langoy</td><td>swim</td><td>plato</td><td>plate</td></tr>
```

File: HTML/Baybayin Words.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+               <tr><td>kayumanggi</td><td>brown</td><td>lipad</td><td>fly</td><td>kutsilyo</td><td>knife</td></tr>
+			   <tr><td>kayamanan</td><td>treasure</td><td>kidlat</td><td>lightning</td><td>kuwintas</td><td>necklace</td></tr>
+			   <tr><td> mahina</td><td>weak</td><td>hapunan</td><td>dinner</td><td>singsing</td><td>ring</td></tr>
+			   <tr><td> bayabas</td><td>guava</td><td>agahan</td><td>breakfast</td><td>hinto</td><td>stop</td></tr>
+			   </table>
```
Comment (samfundev) [inline HTML/Baybayin Words.html:45]:
```suggestion
               <tr><td>kayamanan</td><td>treasure</td><td>kidlat</td><td>lightning</td><td>kuwintas</td><td>necklace</td></tr>
               <tr><td> mahina</td><td>weak</td><td>hapunan</td><td>dinner</td><td>singsing</td><td>ring</td></tr>
               <tr><td> bayabas</td><td>guava</td><td>agahan</td><td>breakfast</td><td>hinto</td><td>stop</td></tr>
               </table>
```

**Review by samfundev — CHANGES_REQUESTED** [review]

**Comment by MrPeanut1028** [comment]
Still need to update the HTML with Sam's suggested changes and the lining up, as well as the SVG Minification.

File: HTML/Baybayin Words.html [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <title>Baybayin Words — Keep Talking and Nobody Explodes Module</title>
+    <meta name="viewport" content="initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+</head>
```
Comment (MrPeanut1028) [inline HTML/Baybayin Words.html:12]: Copy over the header from the template, this isn't the correct one (plus the spacing on line 8 is off.

File: HTML/Baybayin Words.html [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">Baybayin Words</span>
+            </div>
+            <div class="page-content">
+                <h3>Baybayin Alphabet Reference</h3>
+                <img style='width: 100%' src="img/Baybayin Words/Alphabet Table.png" />
+            </div>
+            <div class="page-footer relative-footer">Page 2 of 2</div>
+        </div>
+        </div>
```
Comment (MrPeanut1028) [inline HTML/Baybayin Words.html:60]: I believe there's an extra closing tag here.

**Review by MrPeanut1028 — CHANGES_REQUESTED** [review]

File: HTML/Baybayin Words.html [hunk 11]
```diff
@@ -0,0 +1,66 @@
+﻿<!DOCTYPE html>
+<html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>Module Name — Keep Talking and Nobody Explodes Module</title>
```
Comment (MrPeanut1028) [inline HTML/Baybayin Words.html:7]: Need the module name in the header, not `Module Name`.

**Review by MrPeanut1028 — APPROVED** [review]

**Pull request merged**